src/wallet/rpcwallet.cpp

```diff
@@ -3263,11 +3409,13 @@ UniValue z_getbalance(const UniValue& params, bool fHelp)
     if (fHelp || params.size() == 0 || params.size() > 3)
         throw runtime_error(
             "z_getbalance \"address\" ( minconf inZat )\n"
+            "\nDEPRECATED\n"
             "\nReturns the balance of a taddr or zaddr belonging to the node's wallet.\n"
             "\nCAUTION: If the wallet has only an incoming viewing key for this address, then spends cannot be"
-            "\ndetected, and so the returned balance may be larger than the actual balance.\n"
+            "\ndetected, and so the returned balance may be larger than the actual balance."
+            "\nThe argument address may not be a Unified Address; please use z_getbalanceforaddress instead.\n"
             "\nArguments:\n"
-            "1. \"address\"      (string) The selected address. It may be a transparent or private address.\n"
+            "1. \"address\"        (string) The selected address. It may be a transparent, shielded, or Unified address.\n"
```

Comment:
This edit doesn't match the paragraph above (which says that Unified Addresses are rejected). Revert this change (or at least remove UAs from it).

Reply:
"private" is not the correct term.
```suggestion
            "1. \"address\"        (string) The selected address. It may be a transparent or shielded address.\n"
```